dotnet-authserver/src/TeacherIdentity.AuthServer/Models/Mappings/UserMapping.cs

```diff
@@ -27,6 +27,7 @@ public void Configure(EntityTypeBuilder<User> builder)
         builder.Property(u => u.IsDeleted).IsRequired().HasDefaultValue(false);
         builder.Property(u => u.TrnLookupStatus);
         builder.Property(u => u.MergedWithUserId);
+        builder.Property(u => u.MobileNumber).HasMaxLength(100);
```

Comment:
I think we need an unique index on `MobileNumber` too (with an `is_deleted = false` condition)